Lib/fontTools/cffLib.py

```diff
@@ -1731,17 +1743,16 @@ def xmlRead(self, name, attrs, content, parent):
 	((12, 36),	'FDArray',		'number',	None,	FDArrayConverter()),
 	(17,		'CharStrings',		'number',	None,	CharStringsConverter()),
 	(24,		'VarStore',		'number',	None,	VarStoreConverter()),
-	(25,		'maxstack',		'number',	None,	None),
```

Comment:
This a work-around for a current limitation on my scripts for building OpenType/CFF2 fonts. It is currently convenient to able to convert the default master CFF object to a CFF2 object by doing a bit of field editing and then compiling it. That requires the CFF compiler methods to be able to handle the CFF2 fields. This is messy, and I plan to fix this in the longer run - it is not particularly difficult to do so.